### ISSUE TITLE
Erro ao digitar aspas

### ISSUE THREAD

**Issue by arturbruno17** [PowerShell/PSReadLine#1522]
**Ao digitar aspas duplas ou simples, o PowerShell apresenta um erro.**

```
Últimas 1 Chaves:
 "

Exceção:
System.ArgumentOutOfRangeException: O valor deve ser maior ou igual a zero e menor que o tamanho do buffer do console nessa dimensão.
Nome do parâmetro: left
Valor real era -1.
   em System.Console.SetCursorPosition(Int32 left, Int32 top)
   em Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value) 
   em Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   em `Microsoft.PowerShell.PSConsoleReadLine.ForceRender()`
   em Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   em Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   em Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   em Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
`````

**Comment by iSazonov** [comment]
Please use latest PSReadline version to get the fix.

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
